langserver/ast.go

```diff
@@ -59,7 +59,7 @@ func goRangesToLSPLocations(fset *token.FileSet, nodes []*ast.Ident) []lsp.Locat
 
 func goRangeToLSPLocation(fset *token.FileSet, pos token.Pos, end token.Pos) lsp.Location {
 	return lsp.Location{
-		URI:   "file://" + fset.Position(pos).Filename,
```

Comment:
> We also return triple slash on unix, since fset.Position(pos).Filename starts with /. I'm surprised we can't do this on windows.

On Windows, filename will be `C:/foo/bar` thus making `file://C:/foo/bar` while we need `file:///C:/foo/bar`
Also, I believe that refactoring to introduce common path2uri function is good as is :)

> How easy is it to get a windows environment I can SSH or RDP into? 

I'm using neat feature Appveyor https://www.appveyor.com/docs/how-to/rdp-to-build-worker/ to connect to build server via RDP, you have 1 hour to experiments. Hope this helps

> Relatedly, can't we change vfs.OS so it returns paths that just work?
Won't be enough. `vfs.OS` may return list of directory entries, for example, and then `build.Context` will be responsible for constructing path elements...